### PR TITLE
final clone result, second pull request

### PR DESCRIPTION
**Pull request by JayKowski**
the final youtube clone page

### CODE REVIEW

File: css/main.css
Comment (bolah2009) [inline css/main.css]: `AWESOME`

Nice job using `media query` to make it responsive. :smile: :white_check_mark:

File: index.html
Comment (bolah2009) [inline index.html:13]: `OPTIONAL`

Use semantic HTML elements.

File: index.html
Comment (bolah2009) [inline index.html:307]: `OPTIONAL`

What if you used the `footer` element. I can see you have a `class="footer"` meaning it's the footer, right? :thinking:

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Awesome clone Team :medal_military: :clap: 

The [clone](https://raw.githack.com/JayKowski/html-collaborative-project/developer/index.html) looks good to me, used image and video embedding appropriately. :muscle: 

However, there are a few things the project need to make it look more professional. :worried: See below:

-  [`REQUIRED`] Add linter: See more details [here](https://github.com/JayKowski/html-collaborative-project/pull/1#pullrequestreview-276218813)

-  [`REQUIRED`] Please add a descriptive readme file:  Add the assignment link, the project specification, link to clone page and source page, screenshots and other pieces of information that would describe the project.

> Remember about descriptive README file.`

- [OPTIONAL _but strongly recommended_ ] Use semantic HTML elements as much as possible: Use semantic elements, `<nav>` for navbar, `<main>` for the dominant content of the <body> of a document, `<footer>`  element for the footer content, e.t.c

- [Next Project] Use GitHubFlow: For HTML & CSS projects, use only one feature branch (e.g. `embedding-images-and-video`, `html-forms`, `responsive-design`, etc) and **do not merge any feature branch to master until it's been approved**.

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### Note that comments tagged `OPTIONAL` are strongly recommended but not required to approve the project.

**Review by bolah2009 — COMMENTED** [review]
`REQUIRED`

Add `.stickler.yml` file to the root of the project. Also let's use [this file](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml). :point_down: 

``` yml
linters:
  stylelint:
    config: 'stylelint.config.js'
```
- Also, add the `stylelint.config.js` that was referenced above, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js)

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/css).

**Comment by adaorachi** [comment]
Hi Bola, my partner and I can't seem to correct all errors flagged by linter. We've been on it for several hours without making any progress. Even the linter extensions we installed in our text editors doesn't seem to work well. We'll appreciate it if you can help us out.

**Comment by bolah2009** [comment]
> Hi Bola, my partner and I can't seem to correct all errors flagged by linter. We've been on it for several hours without making any progress. Even the linter extensions we installed in our text editors doesn't seem to work well. We'll appreciate it if you can help us out.

Hi @adaorachi , sorry am just seeing this. Try formatting your code and run  [`stylelint`](https://stylelint.io/) locally. Check [here](https://github.com/microverseinc/linters-config/tree/master/css#troubleshooting) and [here](https://stylelint.io/) for more details.

**Comment by samgaco** [comment]
#### Hi @adaorachi and @JayKowski :vulcan_salute: 

#### Great job with the youtube page, looks brilliant :clap: You are approved. Congratulations :+1: You may merge to the master branch and submit the project completion form :muscle:

**Pull request merged**
